test/jdk/javax/accessibility/manual/TableDemo.html

```diff
@@ -78,4 +78,4 @@
 </body>
 </html>
 
-                                        
\ No newline at end of file
+

```

Comment:
The same as above, trailing white-space caused GitHub to indicate the missing blank line at the end of the file.